contractcourt/channel_arbitrator_test.go

```diff
@@ -187,6 +187,12 @@ func (*mockChainIO) GetBlock(blockHash *chainhash.Hash) (*wire.MsgBlock, error)
 	return nil, nil
 }
 
+func (*mockChainIO) GetBlockHeader(blockHash *chainhash.Hash) (*wire.BlockHeader,
```

Comment:
The linter only looks at new code, so that's why this triggers. Can just remove the argument name (`blockHash`) as we're not using the value anyway. Then everything should fit on one line easily:
```
func (*mockChainIO) GetBlockHeader(*chainhash.Hash) (*wire.BlockHeader, error) {
	return nil, nil
}
```